specification/recoveryservices/resource-manager/readme.md

```diff
@@ -74,3 +74,24 @@ csharp:
   clear-output-folder: true
 ```
 
+## Python
+
+These settings apply only when `--python` is specified on the command line.
+
+```yaml $(python)
+python:
+  azure-arm: true
+  namespace: azure.mgmt.recoveryservices
+  package-version: 0.1.0
+  output-folder: $(output-folder)Generated/Python
+  payload-flattening-threshold: 2
+  license-header: MICROSOFT_MIT_NO_VERSION
+```
+
+## Suppression
+
+``` yaml
+directive:
+  - suppress: DefinitionsPropertiesNamesCamelCase
+    reason: Autorest invalidates two letter acronyms as well and changes in data contracts require service wide changes and require more time

```

Comment:
@sonathan thanks for pointing this out. Regarding the specific violation DefinitionsPropertiesNamesCamelCase the justification in the email and the above doesn't seem to match the reason why the error is being reported. I see properties named as as "Provider" for "ClientDiscoveryDisplay", which violates the rule. I understand this is a breaking change and you may not want to take it right now, would this be a permanent exception? @ravbhatnagar @salameer